kafka/hbase-kafka-proxy/src/main/java/org/apache/hadoop/hbase/kafka/KafkaTableForBridge.java

```diff
@@ -75,106 +73,94 @@ public RegionLocator getRegionLocator() throws IOException {
     throw new UnsupportedOperationException();
   }
 
-  public KafkaTableForBridge(TableName tableName,
-                 Configuration conf,
-                 TopicRoutingRules routingRules,
-                 Producer<byte[],byte[]> producer,
-                 DatumWriter<HBaseKafkaEvent> avroWriter){
-    this.conf=conf;
-    this.tableName=tableName;
-    this.tableAsBytes=this.tableName.toBytes();
-    this.routingRules=routingRules;
-    this.producer=producer;
-    this.avroWriter=avroWriter;
+  public KafkaTableForBridge(TableName tableName, Configuration conf,
+    TopicRoutingRules routingRules, Producer<byte[], byte[]> producer,
+    DatumWriter<HBaseKafkaEvent> avroWriter) {
```

Comment:
One style nit, could we config style like these:
```
public KafkaTableForBridge(TableName tableName, Configuration conf,
    TopicRoutingRules routingRules, Producer<byte[], byte[]> producer,
    DatumWriter<HBaseKafkaEvent> avroWriter) {
 this.conf = conf;
```
add one more indent for a newline after a method declaration

Reply:
Hi @Reidddddd this can be possibly solved but would require changing `hbase_eclipse_formatter.xml`. I have verified same issue exists in hbase main repo as well (as we have just copied `hbase_eclipse_formatter.xml` from there). We can try to fix that but I would say lets do it as a separate change as we may need to rerun spotless for all hbase repos after this change.